src/index.ts

```diff
@@ -142,7 +164,25 @@ program
     )
     .option("-n, --note [note]", "Note attached to the release")
     .action(({ note }: any) => {
-        release(note);
+        const isConfirmed = "Yes";
+        const isDenied = "No, let's do this release!";
+        inquirer
+            .prompt({
+                choices: [isConfirmed, isDenied],
+                message: "Are you releasing a breaking change?",
```

Comment:
Maybe here too, something like: "Non breaking changes will take effect immediately on the stores that have your block installed".

Reply:
I'm not opposed to `You are about to release your updates to <BLOCK_NAME> to production. Non breaking changes will take effect immediately on the stores that have your block installed. Continue y/n?`